src/DataFormatter/XMLDataFormatter.php

```diff
@@ -148,6 +148,8 @@ public function convertDataObjectWithoutHeader(DataObject $obj, $fields = null,
                 if (!singleton($relClass)->stat('api_access')) {
                     continue;
                 }
+                // backslashes in FQCNs kills both URIs and XML
+                $relClass = $this->sanitiseClassName($relClass);
```

Comment:
Seems like this should be below this block? i.e. only sanitise it right before writing the URLs and XML?

```
// Field filtering
if ($fields && !in_array($relName, $fields)) {
    continue;
}
if ($this->customRelations && !in_array($relName, $this->customRelations)) {
    continue;
}
```